person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def can_use_services?
+    @age > 18 || @parent_permission == true
+  end
+
+  private
+
+  def of_age?
+    @age > 18
```

Comment:
- [ ] `@age` should be **greater than or equal** to 18, not only greater than. Kindly fix this.